src/components/molecules/Visualizer/Plugin/context.tsx

```diff
@@ -82,10 +82,16 @@ export type Props = {
   flyToGround: (destination: FlyToDestination, options?: CameraOptions, offset?: number) => void;
 };
 
+type SelectedRearthEventType = Pick<
```

Comment:
```suggestion
type SelectedReearthEventType = Pick<
```

Reply:
I fixed this review in [refactor: rename SelectedRearthEventType](https://github.com/reearth/reearth-web/pull/357/commits/7884c159efb0c928bce18cec9038d678589ec3dc)